receiver/mysqlreceiver/README.md

```diff
@@ -27,6 +27,10 @@ The following settings are optional:
 - `collection_interval` (default = `10s`): This receiver collects metrics on an interval. This value must be a string readable by Golang's [time.ParseDuration](https://pkg.go.dev/time#ParseDuration). Valid time units are `ns`, `us` (or `µs`), `ms`, `s`, `m`, `h`.
 
 - `transport`: (default = `tcp`): Defines the network to use for connecting to the server.
+- `perf_events_statements`: Additional configuration for query to build `mysql.perf.events.statements` and `mysql.perf.events.statements.wait.time` metrics:
```

Comment:
```suggestion
- `statement_events`: Additional configuration for query to build `mysql.statement_events.count` and `mysql.statement_events.wait.time` metrics:
```

I think this would follow, based on the suggested metric names. WDYT?